doc/whats-new.rst

```diff
@@ -10,6 +10,16 @@ What's New
     import xarray as xr
     np.random.seed(123456)
 
+.. _whats-new.0.7.1:
+
+v0.7.1 (TBA)
+------------------------
+
+Enhancements
+~~~~~~~~~~~~
+-xarray version of np.dot :py:meth:`~DataArray.dot`. Takes the sum product over the shared dimensions of two DataArrays. Can be useful for measuring correlation over common dimensions of two DataArrays.
```

Comment:
I don't think you need "Can be useful for..."

Dot products are pretty broadly useful in science :)